test/map/map_builder_test.dart

```diff
@@ -180,6 +180,21 @@ void main() {
           {1: '1', 2: '2'});
     });
 
+    test('has a method like Map.length', () {
+      expect(new MapBuilder<int, String>({1: '1', 2: '2'}).length, 2);
+      expect(new BuiltMap<int, String>({1: '1', 2: '2'}).toBuilder().length, 2);
+      
+      expect(new MapBuilder<int, String>({}).length, 0);
+      expect(new BuiltMap<int, String>({}).toBuilder().length, 0);
+    });
+
+    test('has methods like Map.isEmpty and Map.isNotEmpty', () {
```

Comment:
Please split into separate methods for isEmpty and isNotEmpty as you had for List.